src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,9 @@
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+
+export const PeoplePage = () => (
```

Comment:
Put the fetching logic in this component

<img width="243" alt="image" src="https://github.com/mate-academy/react_people-table-basics/assets/49764881/fde07633-210a-4fda-85bf-fa11bd863076">

Also for pages create a separate `pages` folder on the same level as `components